app/gui/src/controller/graph.rs

```diff
@@ -999,14 +1013,8 @@ impl Handle {
 ///
 /// It just applies the information from the metadata.
 impl span_tree::generate::Context for Handle {
-    fn call_info(&self, id: node::Id, name: Option<&str>) -> Option<CalledMethodInfo> {
-        let db = &self.suggestion_db;
-        let metadata = self.module.node_metadata(id).ok()?;
-        let db_entry = db.lookup_method(metadata.intended_method?)?;
-        // If the name is different than intended method than apparently it is not intended anymore
-        // and should be ignored.
-        let matching = if let Some(name) = name { name == db_entry.name } else { true };
-        matching.then(|| db_entry.invocation_info())
+    fn call_info(&self, _id: node::Id, _name: Option<&str>) -> Option<CalledMethodInfo> {
+        None
     }
 }
```

Comment:
We already have `Empty` context for such cases: https://github.com/enso-org/enso/blob/d8f6f3fd52d87f63d8b067b609162423ed23ebcc/app/gui/language/span-tree/src/generate/context.rs

So, if we are not in some weird generics' machinery, the controller should not implement Context at all, and use `Empty` instead if itself where needed.

Reply:
Done, also removed a couple of tests that relied on  arguments information from node metadata.